homeassistant/components/philips_js/__init__.py

```diff
@@ -116,8 +130,22 @@ def _update_listeners():
             ),
         )
 
+    @property
+    def _notify_wanted(self):
+        """Return if the notify feature should be active.
+
+        We only run it when TV is considered fully on. When powerstate is in standby, the TV
+        will go in low power states and seeminly break the http server in odd ways.
+        """
+        return (
+            self.api.on
+            and self.api.powerstate == "On"
+            and self.api.notify_change_supported
+            and self.options[CONF_ALLOWNOTIFY]
+        )
+
     async def _notify_task(self):
-        while self.api.on and self.api.notify_change_supported:
+        while self._notify_wanted:
```

Comment:
This is a breaking change, right? As it will now all of sudden be disabled by default?
While it previously might have worked and thus be used?

Reply:
It not really breaking anything. It's slowing down updates a bit. Seems most TV's that support his (android tv) crash when in standby with notification service active. These TV's have not been supported before two releases back, but has been crash prone since introduced.

So all features are still there, just that it will take 30 seconds to react. I suppose i could change the default to on and leave the limitation to not use while in standby. It will mitigate the crashiness and leave the speed intact while on.

Reply:
I will do it in two steps instead. Just disable even service while in standby for now. That should be backportable